azure/functions/decorators/function_app.py

```diff
@@ -1373,3 +1374,105 @@ def decorator():
             return decorator()
 
         return wrap
+
+    def read_table(self,
+                   arg_name: str,
+                   connection: str,
+                   table_name: str,
+                   row_key: Optional[str] = None,
+                   partition_key: Optional[str] = None,
+                   data_type: Optional[
```

Comment:
why take and filter are not defined as params here?
<img width="976" alt="image" src="https://user-images.githubusercontent.com/12449837/167010245-b01f6552-724b-4d79-a3cc-0a748bd8eea0.png">

Reply:
updated as per recommendation.